### ISSUE TITLE
Commands

### ISSUE THREAD

**Issue by sindresorhus** [wbond/sublime_terminal#14]
You should add commands to open the Terminal, not everyone like keyboard shortcuts:

```json
[{
    "caption": "Open Terminal in current file location",
    "command": "open_terminal"
}, {
    "caption": "Open Terminal in current project location",
    "command": "open_terminal_project_folder"
}]
```

**Comment by twolfson** [comment]
To be more inline with Sublime conventions, how about:

- Terminal: Open
- Terminal: Open in project folder

**Comment by MauriceZ** [comment]
I submitted a pull request for this here: https://github.com/wbond/sublime_terminal/pull/93

**Comment by twolfson** [comment]
This has been patched/released by @MauriceZ via #93 and released in `1.8.3`
